src/gui/guiFormSpecMenu.cpp

```diff
@@ -3392,12 +3538,10 @@ void GUIFormSpecMenu::acceptInput(FormspecQuitMode quitmode=quit_mode_no)
 							fields[name] = "VAL:" + os.str();
  					}
 				}
-				else
-				{
-					IGUIElement* e = getElementFromId(s.fid);
-					if(e != NULL) {
+				else {
```

Comment:
`} else {`

Reply:
This is adapted to the code style of the whole if else construct here. I would have to change it everywhere.
(Also it's quite readable.)

Reply:
Only needed for touched lines